src/rafts/lighthouse/commands/abstract.js

```diff
@@ -27,10 +28,18 @@ class ImgCommand extends BaseCommand {
 
     context.fillStyle = '#57C7FF';
     context.fillRect(0, 0, width, height);
-    const attachment = new Discord.MessageAttachment(canvas.toBuffer(), 'welcome-image.png');
+    var i;
+    var maxIterations = 20;
+    for (i = 0; i < maxIterations; i++) {
```

Comment:
```suggestion
    const maxIterations = 20;
    for (const i = 0; i < maxIterations; i++) {
```
Condense this down, also, might consider taking maxIterations as an argument for this command?

No vars!!!!